tests/clipboard.spec.ts

```diff
@@ -1373,7 +1373,9 @@ test(scoped`clipboard copy multi selection`, async ({ page }) => {
   await focusRichText(page, 1);
   await pasteByKeyboard(page);
   await waitNextFrame(page);
-  await assertRichTexts(page, ['abc', 'defbc', 'd']);
+  await type(page, 'cursor');
+  await waitNextFrame(page);
+  await assertRichTexts(page, ['abc', 'defbc', 'dcursor']);
 });
 
 test.skip(scoped`clipboard copy nested items`, async ({ page }) => {
```

Comment:
Can we recheck the skipped cases? Either re-enable them, remove them or fix them.